Description: Hard-code GSL CBLAS paths and keep gslcblas in gsl_libs
 Debian does not provide a pkg-config file for libgslcblas,
 so just hard-code the paths here for Sage
 .
 Also, when gslcblas is removed from gsl_libs we currently get this error
 at the beginning of the doc build:
 ImportError: /usr/lib/x86_64-linux-gnu/libgsl.so.19: undefined symbol: cblas_ctrmv
 Does libgsl need to be linked against gslcblas?
Author: Ximin Luo <infinity0@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/src/module_list.py
+++ b/sage/src/module_list.py
@@ -12,10 +12,9 @@
 import pkgconfig
 
 # CBLAS can be one of multiple implementations
-cblas_pc = pkgconfig.parse('cblas')
-cblas_libs = cblas_pc['libraries']
-cblas_library_dirs = cblas_pc['library_dirs']
-cblas_include_dirs = cblas_pc['include_dirs']
+cblas_libs = [u'gslcblas', u'm']
+cblas_library_dirs = []
+cblas_include_dirs = []
 
 # TODO: Remove Cygwin hack by installing a suitable cblas.pc
 if os.path.exists('/usr/lib/libblas.dll.a'):
@@ -42,9 +41,9 @@
 # GNU Scientific Library
 # Note we remove the built-in gslcblas
 # The above cblas should already be in the list thanks to #20646
+# On Debian linking to libgsl currently requires also -lgslcblas so we do not remove it.
 gsl_pc = pkgconfig.parse('gsl')
 gsl_libs = gsl_pc['libraries']
-gsl_libs.remove('gslcblas')
 gsl_library_dirs = gsl_pc['library_dirs']
 gsl_include_dirs = gsl_pc['include_dirs']
 
--- a/sage/src/sage/misc/cython.py
+++ b/sage/src/sage/misc/cython.py
@@ -30,17 +30,16 @@
 
 
 # CBLAS can be one of multiple implementations
-cblas_pc = pkgconfig.parse('cblas')
-cblas_libs = list(cblas_pc['libraries'])
-cblas_library_dirs = list(cblas_pc['library_dirs'])
-cblas_include_dirs = list(cblas_pc['include_dirs'])
+cblas_libs = [u'gslcblas', u'm']
+cblas_library_dirs = []
+cblas_include_dirs = []
 
 # TODO: Remove Cygwin hack by installing a suitable cblas.pc
 if os.path.exists('/usr/lib/libblas.dll.a'):
     cblas_libs = 'gslcblas'
 
 standard_libs = [
-    'mpfr', 'gmp', 'gmpxx', 'stdc++', 'pari', 'm', 
+    'mpfr', 'gmp', 'gmpxx', 'stdc++', 'pari', 'm',
     'ec', 'gsl',
 ] + cblas_libs + [
     'ntl']
